WordPress/src/main/res/layout/login_email_password_screen.xml

```diff
@@ -1,126 +1,72 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-
-    <include
-        android:id="@+id/toolbar"
-        layout="@layout/toolbar_login" />
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingLeft="@dimen/margin_extra_large"
+    android:paddingStart="@dimen/margin_extra_large"
+    android:paddingRight="@dimen/margin_extra_large"
+    android:paddingEnd="@dimen/margin_extra_large"
+    android:layout_marginBottom="@dimen/margin_extra_large">
+
+    <TextView
+        style="@style/LoginTheme.TextLabel"
+        android:id="@+id/label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:layout_marginBottom="@dimen/margin_extra_large"
+        tools:text="@string/enter_site_address" />
 
-    <ScrollView
+    <LinearLayout
```

Comment:
This `LinearLayout` can be optimized by using `RelativeLayout` and removing the nested `LinearLayout` as shown below.

```xml
    <RelativeLayout
        android:layout_height="wrap_content"
        android:layout_width="match_parent"
        android:gravity="center_vertical" >

        <ImageView
            android:id="@+id/login_email_icon"
            android:contentDescription="@string/login_email_image"
            android:layout_centerVertical="true"
            android:layout_height="wrap_content"
            android:layout_marginEnd="@dimen/margin_extra_large"
            android:layout_marginRight="@dimen/margin_extra_large"
            android:layout_width="wrap_content"
            android:tint="@color/grey_lighten_10"
            app:srcCompat="@drawable/ic_user_grey_24dp" >
        </ImageView>

        <TextView
            android:id="@+id/login_email_label"
            android:layout_height="wrap_content"
            android:layout_toEndOf="@id/login_email_icon"
            android:layout_toRightOf="@id/login_email_icon"
            android:layout_width="match_parent"
            android:text="@string/email_address"
            style="@style/LoginTheme.InputLabelStatic" >
        </TextView>

        <TextView
            android:id="@+id/login_email"
            android:layout_below="@id/login_email_label"
            android:layout_height="wrap_content"
            android:layout_toEndOf="@id/login_email_icon"
            android:layout_toRightOf="@id/login_email_icon"
            android:layout_width="match_parent"
            tools:text="s@b.com"
            style="@style/LoginTheme.TextLabel" >
        </TextView>

    </RelativeLayout>
```